app/services/db/monthly_sp_auth_count/unique_monthly_auth_counts_by_partner.rb

```diff
@@ -93,23 +93,51 @@ def call(key:, issuers:, start_date:, end_date:)
       # @return [Array<String>]
       def build_queries(issuers:, months:)
         months.map do |month_range|
+          today = Date.today
           params = {
             range_start: month_range.begin,
             range_end: month_range.end,
             year_month: month_range.begin.strftime('%Y%m'),
             issuers: issuers,
+            one_years_ago: today - 365,
+            two_years_ago: today - 2 * 365,
+            three_years_ago: today - 3 * 365,
+            four_years_ago: today - 4 * 365,
+            five_years_ago: today - 5 * 365,
           }.transform_values { |value| quote(value) }
 
+
+
           format(<<~SQL, params)
             SELECT
               sp_return_logs.user_id
             , %{year_month} AS year_month
             , COUNT(sp_return_logs.id) AS auth_count
             , sp_return_logs.ial
+            , SUM (
+              CASE WHEN sp_return_logs.profile_verified_at IS BETWEEN %{now) AND %{one_years_ago}
+              THEN 1 ELSE 0 END) as partner_ial2_new_unique_users_year1
+            , SUM (
+              CASE WHEN sp_return_logs.profile_verified_at IS BETWEEN %{one_years_ago) AND %{two_years_ago}
+              THEN 1 ELSE 0 END) as partner_ial2_new_unique_users_year2
+            , SUM (
+              CASE WHEN sp_return_logs.profile_verified_at IS BETWEEN %{two_years_ago) AND %{three_years_ago}
+              THEN 1 ELSE 0 END) as partner_ial2_new_unique_users_year3
```

Comment:
As a style thing, I think it's clearer when:
1. we capitalize SQL key words (`AS`)
2. we indent inside parens

```suggestion
            , SUM (
                CASE
                WHEN sp_return_logs.profile_verified_at IS BETWEEN %{two_years_ago) AND %{three_years_ago}
                THEN 1
                ELSE 0
                END
              ) AS partner_ial2_new_unique_users_year3
```

See this report for an example format here:
https://github.com/18F/identity-idp/blob/main/app/services/db/identity/sp_active_user_counts.rb#L22-L27

I know this gets repeated a lot but I think it's easier to understand what each line does this way

Reply:
This was addressed